server.js

```diff
@@ -2,33 +2,41 @@ require("dotenv").config();
 const express = require("express");
 const app = express();
 const cors = require("cors");
+const path = require("path");
 const corsOptions = require("./config/corsOptions");
 const credentials = require("./middleware/credentials");
 const mongoose = require("mongoose");
 const connectDB = require("./config/dbConn");
 const errorHandler = require("./middleware/errorHandler.js");
 
 const PORT = process.env.PORT || 3500;
+const env = process.env.NODE_ENV || "dev";
 
 // connecting DB
-connectDB();
+if (env !== "dev")
+    connectDB();
```

Comment:
Should we not connect to DB for dev env? Should we make another DB model for the dev?

Reply:
Because we will need to test in dev env only